sdk/storage/azure-storage-file-datalake/tests/test_file_system.py

```diff
@@ -5,6 +5,8 @@
 # Licensed under the MIT License. See License.txt in the project root for
 # license information.
 # --------------------------------------------------------------------------
+import uuid
```

Comment:
Looks like this is left over in the diff :) Similar thing in async.

Reply:
Good catch! I imported this in `test_file_system` but in the async version apparently there is a test that uses `uuid` and SAS tokens (marked as live only, so they won't run into the same error I ran into) 😄